vita.header_warn.cmake

```diff
@@ -48,6 +48,13 @@ endif()
 math(EXPR __vitasdk_expires_wrong_libs_module_name "20240101 + 2 * 100")
 
 if(${__vitasdk_current_time} LESS ${__vitasdk_expires_wrong_libs_module_name})
+endif()
+
+if(Off)
```

Comment:
hm... how devs read this message if you move this block to off part?

Reply:
Disabled the message by correcting the module name without changing the lib name.